src/lib/styles.css

```diff
@@ -168,4 +174,48 @@
 .react-quiz-container .questionBtnContainer{
   display:flex;
   justify-content: space-between;
+}
+
+/* Mobile Responsive */
+@media screen and (max-width: 767px) {
```

Comment:
please stick to Mobile First Principle. No `max-width` and `and` please.

Reply:
hey, I Understand the Mobile First Principle part and act on it , but why not No `max-width` and `and` I don't understand can you explain it to me please.